### ISSUE TITLE
[deconz] no switches or sensors discovered

### ISSUE THREAD

**Issue by olemr** [openhab/openhab-addons#5617]
![image](https://user-images.githubusercontent.com/28068256/57969211-db10f680-7974-11e9-913a-c45d457ab5f5.png)
Running on x64 Ubuntu Server.
![image](https://user-images.githubusercontent.com/28068256/57969223-157a9380-7975-11e9-9aec-851006a675c7.png)
LogoYARC! v1.1.2
 
```
http://192.168.1.6/api/6B9F7BD8C9/sensors
Payload:
{"devicetype": "SV74" }
Custom Headers
Authentication
Response
200
Request URL: http://192.168.1.6/api/6B9F7BD8C9/sensors
Request Method: GET
Response Time: 0.027 seconds
Response Status: 200 - OK
Response Body
Response Body (RAW)
Response Headers
Request Details
 {
  "1": {
    "config": {
      "configured": true,
      "on": true,
      "sunriseoffset": 30,
      "sunsetoffset": -30
    },
    "etag": "6964c50a6a3c22bdc0aada60b5e09a3b",
    "manufacturername": "Philips",
    "modelid": "PHDL00",
    "name": "Daylight",
    "state": {
      "dark": false,
      "daylight": true,
      "lastupdated": "2019-05-18T11:47:03",
      "status": 170
    },
    "swversion": "1.0",
    "type": "Daylight",
    "uniqueid": "00:21:2e:ff:ff:02:2c:85-01"
  },
  "2": {
    "config": {
      "battery": 91,
      "on": true,
      "reachable": true,
      "temperature": 2200
    },
    "ep": 2,
    "etag": "6964c50a6a3c22bdc0aada60b5e09a3b",
    "manufacturername": "LUMI",
    "mode": 1,
    "modelid": "lumi.sensor_cube",
    "name": "Pink TV Cube",
    "state": {
      "buttonevent": 7000,
      "lastupdated": "2019-05-18T11:25:15"
    },
    "swversion": "20160407",
    "type": "ZHASwitch",
    "uniqueid": "00:15:8d:00:01:01:69:58-02-0012"
  },
  "3": {
    "config": {
      "battery": 91,
      "on": true,
      "reachable": true,
      "temperature": 2200
    },
    "ep": 3,
    "etag": "6964c50a6a3c22bdc0aada60b5e09a3b",
    "manufacturername": "LUMI",
    "mode": 1,
    "modelid": "lumi.sensor_cube",
    "name": "Pink TV Cube",
    "state": {
      "buttonevent": -3571,
      "lastupdated": "2019-05-18T11:25:16"
    },
    "swversion": "20160407",
    "type": "ZHASwitch",
    "uniqueid": "00:15:8d:00:01:01:69:58-03-000c"
  },
  "4": {
    "config": {
      "battery": 100,
      "on": true,
      "reachable": true,
      "temperature": 2300
    },
    "ep": 2,
    "etag": "a8e824e4005e032a039a677fee8e617e",
    "manufacturername": "LUMI",
    "mode": 1,
    "modelid": "lumi.sensor_cube.aqgl01",
    "name": "White Radio Cube",
    "state": {
      "buttonevent": 7000,
      "lastupdated": "2019-05-18T09:02:40"
    },
    "swversion": "20160704",
    "type": "ZHASwitch",
    "uniqueid": "00:15:8d:00:02:31:0e:e0-02-0012"
  },
  "5": {
    "config": {
      "battery": 100,
      "on": true,
      "reachable": true,
      "temperature": 2300
    },
    "ep": 3,
    "etag": "a8e824e4005e032a039a677fee8e617e",
    "manufacturername": "LUMI",
    "mode": 1,
    "modelid": "lumi.sensor_cube.aqgl01",
    "name": "White Radio Cube",
    "state": {
      "buttonevent": 1223,
      "lastupdated": "2019-04-25T20:01:28"
    },
    "swversion": "20160704",
    "type": "ZHASwitch",
    "uniqueid": "00:15:8d:00:02:31:0e:e0-03-000c"
  },
  "6": {
    "config": {
      "battery": 100,
      "group": "9569",
      "on": true,
      "reachable": true
    },
    "ep": 2,
    "etag": "96b30db72cb610e36b4a6bef0c125c08",
    "manufacturername": "Philips",
    "mode": 1,
    "modelid": "RWL021",
    "name": "Dimmer Switch Kitchen",
    "state": {
      "buttonevent": 1002,
      "lastupdated": "2019-05-18T11:25:04"
    },
    "swversion": "5.45.1.17846",
    "type": "ZHASwitch",
    "uniqueid": "00:17:88:01:04:e7:c1:26-02-fc00"
  }
}

``` 

![image](https://user-images.githubusercontent.com/28068256/57969254-7b671b00-7975-11e9-8ac7-ed3c1101ce07.png)

![image](https://user-images.githubusercontent.com/28068256/57969230-32af6200-7975-11e9-8105-afd62824f23d.png)

Nothing appears in the Inbox: (also tried manual scan)
![image](https://user-images.githubusercontent.com/28068256/57969290-c5e89780-7975-11e9-8e92-1ec88799b18d.png)

Hue lights are OK, but no switches or sensors listed:

 
![image](https://user-images.githubusercontent.com/28068256/57969302-e6b0ed00-7975-11e9-9c0e-1723879f7e33.png)

Let me know if you need more info.

**Comment by cweitkamp** [comment]
Should be fixed with #5444 (11th of April) which is part of snapshot 1568. Snapshot 1549 has been built on 5th of March. I tested it with latest build and new devices popped up immediately in my inbox.

@olemr Can you confirm that?

**Comment by cweitkamp** [comment]
Closed by #5444 .
